scripts/setup-kind.sh

```diff
@@ -45,6 +45,7 @@ if [ -z "${CI_ENABLED}" ]; then
 helm repo add someengineering https://someengineering.github.io/helm-charts
 helm install --namespace resoto resoto someengineering/resoto --set image.tag=$IMAGE_TAG -f - <<EOF
   resotocore:
+    extraArgs: ["--analytics-opt-out"]
```

Comment:
Can we make this a configurable param in the chart? Or do we want every user to have analytics?

Reply:
The value is already configurable. 
The change here is to not gather any metrics for the test runs.

Reply:
I was saying to have explicit Value for this analytics in the chart to disable it. I would believe things like this should be single touch configurable via the chart values without going to resoto yaml & edit.

Reply:
We deliberately moved this decision to configuration time, not installation time.
The user should be able to opt-in / opt-out at any time, regardless of installation.

Reply:
Understood, makes more sense